src/main/java/seedu/duke/Duke.java

```diff
@@ -67,7 +72,7 @@ public String readCommand(){
      */
     private CommandResult executeCommand(Command command) {
         try {
-            CommandResult result = command.execute();
+            CommandResult result = command.execute(items,myBudget);
```

Comment:
Just to clarify if we add in the budget as a parameter for the execute function this means that even commands like "add", "edit", "delete", "mark"  will have to handle the budget too right?  😄 

Is it possible to set the default value of "0" for when the program first runs then from there allow the user to change their budget as and when they like through a command?  🤔"